### PR TITLE
FHIR-38664 - Add profile of Task requesting launch of SMART app

### PR DESCRIPTION
**Pull request by bvdh**
See https://jira.hl7.org/browse/FHIR-38664

### CODE REVIEW

File: input/fsh/task.fsh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Id: smart-launch-types
+Title: "Codes for tasks to application launches"
+Description: "Defines codes for Tasks that request launch of SMART applications."
+* SmartOnFhirCodes#launch-app-ehr "Launch application using the SMART EHR launch"
+* SmartOnFhirCodes#launch-app-standalone "Launch application using the SMART standalone launch"
+
+ValueSet: SmartLaunchInformation
+Id: smart-launch-info
+Title: "Codes for tasks to application launches"
+Description: "Defines codes for Tasks that request launch of SMART applications."
```
Comment (jmandel) [inline input/fsh/task.fsh:47]:
```suggestion
Description: "Defines Launch Information codes for Tasks that request launch of SMART applications."

```

File: input/images/example-authorization-jwks-and-signatures.ipynb [hunk 2]
```diff
@@ -6,7 +6,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# To create a markdown of this notebook, run: jupyter nbconvert --to markdown authorization-example-jwks-and-signatures.ipynb\n",
+    "# To create a markdown of this notebook, run: jupyter nbconvert --to markdown example-authorization-jwks-and-signatures.ipynb\n",
```
Comment (jmandel) [inline input/images/example-authorization-jwks-and-signatures.ipynb:9]: I think the changes to these python examples aren't related to Task 38664. Could you submit these changes in a separate PR?

File: input/pages/task-launch.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+### Task profile for launch SMART applications (EXPERIMENTAL)
+
+
+This section defines a set of profiles of the FHIR Task resource that requests launch of a SMART application. These tasks could be used to instruct staff to run an application or as a result of a Clinical Reasoning deployment.
```
Comment (jmandel) [inline input/pages/task-launch.md:4]:
```suggestion
This section defines a set of profiles of the FHIR Task resource that requests launch of a SMART application. These tasks could be used to recommend an application for staff to launch, e.g. as a result of a Clinical Reasoning deployment.

```

File: input/pages/task-launch.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+Two profiles are defined:
+
+* [task-ehr-launch](StructureDefinition-task-ehr-launch.html), requests an EHR launch with optional appContext.
+* [task-standalone-launch](StructureDefinition-task-standalone-launch.html), requests a standalone launch.
+
+#### Requesting an EHR launch
+
+A Task with the profile [task-ehr-launch](StructureDefinition-task-ehr-launch.html), requests an EHR launch with optional appContext.
```
Comment (jmandel) [inline input/pages/task-launch.md:13]:
```suggestion
A Task with the profile [task-ehr-launch](StructureDefinition-task-ehr-launch.html), requests an EHR launch with optional `appContext`.

```

File: input/pages/task-launch.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+The subject field, if present indicates the patient resource to be used in the launch context.
```
Comment (jmandel) [inline input/pages/task-launch.md:15]:
```suggestion
The `Task.for` field, if present indicates the Patient resource to be used in the launch context.

```

I think this correction is consistent with the intent + example below?

File: input/pages/task-launch.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+the encounter field, if present indicates the encounter reource to be used in the launch context.
```
Comment (jmandel) [inline input/pages/task-launch.md:17]:
```suggestion
the `Task.encounter` field, if present indicates the Encounter resource to be used in the launch context.

```

File: input/images/example-authorization-jwks-and-signatures.ipynb (hunk 2 above)
Comment (bvdh) [reply input/images/example-authorization-jwks-and-signatures.ipynb:9]: Sure, if they are needed. Are they?

File: input/pages/task-launch.md (hunk 5 above)
Comment (bvdh) [reply input/pages/task-launch.md:17]: Correct

File: input/images/example-authorization-jwks-and-signatures.ipynb (hunk 2 above)
Comment (jmandel) [reply input/images/example-authorization-jwks-and-signatures.ipynb:9]: The examples are needed in the spec, but they're not related to this PR. I see there was another PR (https://github.com/HL7/smart-app-launch/pull/390) to introduce changes, and an outstanding question from @Healthedata1 on that PR.

Comment (bvdh) [reply input/images/example-authorization-jwks-and-signatures.ipynb:9]: Likely, it was a leftover from the other PR. Do I understand correctly, that we can remove them?

Comment (jmandel) [reply input/images/example-authorization-jwks-and-signatures.ipynb:9]: We cannot remove the examples from the spec. This PR should not change the examples, since they're not mentioned in the resolution of FHIR-38664
